sql/init.sql

```diff
@@ -1,5 +1,17 @@
+DROP SCHEMA SWT;
+
 create schema if not exists SWT;
 
+create table if not exists SWT.users
```

Comment:
Either remove the SWT.user SCHEMA below, or adjust it with this one. No need to have 'dead' code that isn't even outcommented.